ui/admin/app/components/worker-diagram/index.hbs

```diff
@@ -1,6 +1,17 @@
-{{yield
-  (hash
-    SingleFilter=(component 'worker-diagram/single-filter')
-    DualFilter=(component 'worker-diagram/dual-filter')
-  )
-}}
\ No newline at end of file
+<div class='worker-wrapper'>
```

Comment:
Is this necessary?  If so, why?

Reply:
If this is necessary, can the class name be `worker-diagram`?

Reply:
I completely forgot about the weird height issues we had and that is why the original `worker-wrapper` class was added. It has to be added back otherwise we get staggered icons. As you can see below, the HCP item is higher than the other items. Pushing a correction to my refactor with better naming for what it is used for.
<img width="339" alt="Screen Shot 2023-02-08 at 12 53 14 PM" src="https://user-images.githubusercontent.com/29386339/217625424-231e036e-7c9b-4cc7-811a-1fc4e88daeda.png">

Reply:
No longer relevant to this PR